mParticle-Kochava/MPKitKochava.m

```diff
@@ -274,13 +268,11 @@ - (void)start {
             KVATracker.shared.appTrackingTransparency.authorizationStatusWaitTimeInterval = [self.configuration[kvWaitIntervalATT] integerValue];
         }
     }
-    
-    [KVAAdNetworkProduct.shared register];
-    
+        
     [KVATracker.shared startWithAppGUIDString:self.configuration[kvAppId]];
     
     if (self.configuration[kvLimitAdTracking]) {
-        KVATracker.shared.appLimitAdTrackingBool = [self.configuration[kvLimitAdTracking] boolValue] ? TRUE : FALSE;
+        KVATracker.shared.appLimitAdTracking.boolean = [self.configuration[kvLimitAdTracking] boolValue] ? TRUE : FALSE;
```

Comment:
```suggestion
        KVATracker.shared.appLimitAdTracking.boolean = [self.configuration[kvLimitAdTracking] boolValue];
```
Since this already returns a boolean, we can remove the ternary operator. 

I know this was from the old code, but since we're updating this line anyway I figured we might as well remove the unnecessary logic.

Reply:
Unless, are `TRUE` and `FALSE` something Kochava specific? Or were they meant to be `YES` and `NO`? If they're something Kochava specific then I guess we need the ternary after all, but if not, we should remove them.